src/database/organizationSchema.ts

```diff
@@ -0,0 +1,26 @@
+import { Schema, model, models } from "mongoose";
+
+const organizationSchema = new Schema({
+  name: { type: String, required: [true, "Organization name is required."] },
+  description: {
+    type: String,
+    required: [true, "Organization description is necessary."],
+  },
+  website: { type: String, required: false },
+  clerkUser: {
+    type: String,
+    required: [true, "Organization clerk username required."],
+  },
+  logo: { type: String, required: false },
+  reimbursements: {
+    type: [Schema.Types.ObjectId],
```

Comment:
i believe you can change this to `[{ type: Schema.Types.ObjectId, ref: "Reimbursement" }]` to specify the type of schema being referenced